recipes/powershell5.rb

```diff
@@ -23,7 +23,7 @@
 case node['platform']
 when 'windows'
 
-  if ::Windows::VersionHelper.nt_version(node) >= 6.1
+  if ::Windows::VersionHelper.nt_version(node) >= 6.1 && ::Windows::VersionHelper.nt_version(node) < 10
```

Comment:
This shouldn't really matter - since the not_if should keep the resource from attempting to install, but either way is fine...